src/main/java/BudgetTracker.java

```diff
@@ -100,11 +100,15 @@ private void updateOpportunity() throws SQLException, ClassNotFoundException {
         if (updateOppOption == AllMenuOptions.NAME) {
             String newName = name();
             String columnName = "name";
-            databaseCommunicator.updateOpportunity(oldOpportunity, columnName, newName);
+            databaseCommunicator.updateOpportunityStringField(oldOpportunity, columnName, newName);
         } else if (updateOppOption == AllMenuOptions.DESCRIPTION) {
```

Comment:
Would a switch work better here that calls functions for each one of these?

Reply:
I have updated it and I think it works better!